### ISSUE TITLE
gh-pages

### ISSUE THREAD

**Issue by ca1tlin22** [ca1tlin22/responsive-infographic#2]
Hi @thomasjbradley

Just wondering if I changed the branch from master to gh-pages — I can't find a link in settings/anywhere to make sure it works. I haven't been changing the branches for any of my assignments because I didn't realize we had to, I thought the repositories that we cloned from had the default set to gh-pages already.  

Thanks for your time,
Caitlin Gagné

**Comment by thomasjbradley** [comment]
Yeah, they are all set up properly with `gh-pages`. I forgot to do this one and didn't notice until a few people had already cloned it—you unfortunately were one of those people.

Did you create a new branch named `gh-pages`?

**Comment by ca1tlin22** [comment]
I think so — in my desktop Github thing it says my default branch is gh-pages

**Comment by thomasjbradley** [comment]
I guess I'll have to look at it in class, on line it's pointing to something else.

**Comment by ca1tlin22** [comment]
gh-pages fixed!
